db/schema.rb

```diff
@@ -231,11 +231,8 @@
     t.string   "cached_student_class",      limit: 255
     t.datetime "cache_updated"
     t.datetime "waiver_start"
-    t.integer  "phone_carrier_id",          limit: 4
```

Comment:
It looks like some of the phone carrier schema changes leaked into this diff, in the future can we keep these separate?

Reply:
This might be because the delete-phone-carriers branch was pushed before the migrations were run, so the schema hadn't changed yet. Will be more vigilant in the future.